docs/QUICK_START.md

```diff
@@ -0,0 +1,202 @@
+- These 900 MHz band modules were designed specifically for mLRS and arrive pre-flashed with mLRS version 1.3.
+- This option should provide the longest range of the three discussed here.
+- The mR900-30-TX module kit includes an HC04 module for a convenient Bluetooth serial connection to your ground station software running on a computer or Android device.
+- These modules are especially easy to flash via DFU over their USB C connection.
+- These use the SX126x LoRa chip and support 19 Hz, 31 Hz and FSK(50 Hz) modes.
+- This option does not work with any ELRS receivers.
+- The Tx module kit does not include an antenna or a JR bay case which must be 3D printed.  If you don't have access to a 3D printer, you can purchase a print from a number of 3D printing services.
+- The [mR900-22](https://www.mateksys.com/?portfolio=mr900-22) is a compatible lower power receiver which might be appropriate for smaller vehicles.  But it is not yet recommended for new users because it does not include USB for flashing and improvements to ArduPilot serial passthrough to support firmware update without disconnecting the receiver from the flight controller are not yet released.
+
+### MatekSys mLRS [mR24-30-TX](https://www.mateksys.com/?portfolio=mr24-30-tx) module kit with their [mr24-30](https://www.mateksys.com/?portfolio=mr24-30)) receiver or any [Selected ELRS 2.4 GHz receiver](ELRS_RECEIVERS.md#selected-elrs-receivers)
+- These 2.4 GHz band modules were also designed specifically for mLRS and arrive pre-flashed with mLRS version 1.3.
+- The mR24-30-TX module also includes the HC04 module but you may notice some interference since Bluetooth also uses the 2.4 GHz band.
+- These modules are also especially easy to flash via DFU over their USB C connection.
+- 2.4 GHz is a wider band which might support more simultaneous users.
+- These modules support 19 Hz, 31 Hz, 50, and FLRC(111 Hz) modes.
+- The Tx module kit does not include an antenna or JR bay case which must be 3D printed.  If you don't have access to a 3D printer, you can purchase a print from a number of 3D printing services.
+- If you don't need quite as much range, you can opt for one of the smaller and lighter [Selected ELRS 2.4 GHz receivers](ELRS_RECEIVERS.md#selected-elrs-receivers) such as the RadioMaster RP4TD.
+- Some AIO flight controller boards include a low power 2.4 GHz ELRS serial receiver which can also be used with this Tx module.
+
+### FrSky [R9M Tx module](FRSKY_R9.md#r9m-tx-module) with the [Bayck 915M Nano Pro](ELRS_RECEIVERS.md#selected-elrs-receivers) 900 MHz ELRS receiver
+- This option is not highly recommended for new users as it is more complicated and slightly less flexible, but is included here since it allows use of ELRS receivers you may already have.
+- This Tx module uses the SX1276 LoRa chip and supports only 19 Hz mode.  It works with many 900 MHz ELRS receivers.
```

Comment:
"... ELRS receivers, but is incompatible with the MatekSys gear mentioned before."

Reply:
I already mentioned these three options are all mutually incompatible, but OK.  It can't hurt to mention it again.

Reply:
these are ALL just SUGGESTIONS ... if you don't like, ignore :)